components/server/src/user/user-service.ts

```diff
@@ -260,6 +263,13 @@ export class UserService {
             // Return the user's explicit attribution ID.
             return await this.validateUsageAttributionId(user, user.usageAttributionId);
         }
+        if (user?.additionalData?.isMigratedToTeamOnlyAttribution) {
+            const teams = await this.teamDB.findTeamsByUser(user.id);
+            if (teams.length > 0) {
+                return AttributionId.create(teams[0]);
```

Comment:
This means that the firstteam would always be the one to which we attribute. Is the `findTeamsByUser` stable? What happens if I delete my "first" team?

Reply:
I think this is an intermediate "good enough" solution, because for migrated users `user.usageAttributionId` is never undefined. 
In follow-up PRs, I'd like to get rid of the account selector logic for migrated users and have a more sticky and visible selection of the current team, which we would use for attribution.

Reply:
I think it make sense to give a loud signal that this is happening (log.error) and add this context in a comment. :100: